assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js

```diff
@@ -46,7 +46,7 @@ class SearchConsoleDashboardWidgetSiteStats extends Component {
 	setOptions() {
 		const { selectedStats, series, vAxes } = this.props;
 
-		const pageTitle = '' === googlesitekit.pageTitle ? '' : sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHtmlEntity( googlesitekit.pageTitle ) );
+		const pageTitle = googlesitekit.pageTitle && googlesitekit.pageTitle.length ? sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHtmlEntity( googlesitekit.pageTitle ) ) : '';
```

Comment:
This should still have a value when there is no page title, e.g. just `__( 'Search Traffic Summary', 'google-site-kit' )`.